client/src/app/shared/users/user.service.ts

```diff
@@ -10,6 +10,10 @@ import { SortMeta } from 'primeng/api'
 import { BytesPipe } from 'ngx-pipes'
 import { I18n } from '@ngx-translate/i18n-polyfill'
 import { UserRegister } from '@shared/models/users/user-register.model'
+import { User as UserType } from './user.model'
```

Comment:
Why did you use an alias for `User`?

Reply:
Because there is a different `User` above.

Reply:
Then we need to find a name to differentiate the two types. You could rename the first `User` to `UserServerModel` as we do in other files

Reply:
:+1: